minet/instagram/api_scraper.py

```diff
@@ -49,41 +138,196 @@ def __init__(self, cookie="firefox"):
             raise InstagramInvalidCookieError
 
         self.cookie = cookie
+        self.magic_token = None
+        self.nb_calls = 0
```

Comment:
If you rely on a single `nb_calls` property and use modulo operations to assess when you need to act, it will be leaner.